### ISSUE TITLE
Page break locators definition should be revised

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#1709]
When responding to #1644 I became concerned with the definition of Page break locators:
> **page break locators**
programmatic markers that are arranged in a meaningful sequence to determine the location of a page in relation to others in the set.
Examples would be:
>- A digital version of an ebook that has a print version, it includes the page break locators to align with the print edition.
>- A digital book is published with no print equivalent and page break locators are inserted which supports direct navigation across platforms and form factors.

I'm concerned about the use of the word "programmatic". To me, a primary scenario the ePub folks were trying to address is where page numbers are visible in a long document, but there is no mechanism to reach them. In such a scenario, the most likely situation is that the page numbers are just text without a programmatic indicator (probably separated by white space from the text preceding and following them.

What this SC should do is require authors to make such page numbers programmatically indicated and provide a mechanism to them. If we don't remove "programmatic", it seems to make this SC pretty flimsy. I also think some of the wording for the definition is lacking. I suggest:
> **page break locators**
 markers that are arranged in a meaningful sequence to determine fixed points in a document or set of documents.
Examples would be:
>- A digital version of a book includes page break locators to align with a print edition.
>- A digital book is published with no print equivalent; markers are inserted which support direct navigation across platforms and form factors.

**Comment by alastc** [comment]
Hi @mbgower,

Maybe this has been lost in edits, but the SC was formulated so say effectively: **_If_** you add these programmatic markers for pages breaks, then add navigation to them.

There was another SC proposed (that bit the dust early on) trying to get people to add these if you have other editions, but offline is pretty much out of scope for us. 

The primary situation the ePub folks were trying to address is where these exist but the user-agent does not provide the navigation: predominantly for in-browser representations of ePub docs.

@DavidMacDonald am I mis-remembering?

**Comment by mraccess77** [comment]
Programmatically determinable is needed or otherwise this SC would be pretty useless.  Programmatically determinable would allow assistive technology to jump to the page breaks.  If the page breaks were just indicated visually but not programmatically they would not allow users of assistive technology to quickly navigate to them.

**Comment by alastc** [comment]
@mraccess77 I think it's the other way around: If any marker in a sequence for a 'fixed' part of a doc were included, that's a very wide scope.

Such a wide scope could trigger quite a few things requiring navigation. E.g. Headings? Horizontal Rules? 

Anything in the flow of a document (i.e. always below one section / paragraph and above another) could qualify. 

Making it 'programmatic' markers adds the intent for this SC, and narrows the scope.

**Comment by mraccess77** [comment]
@alastc I'm confused as I thought it was only scoped to page break - markers communicating page breaks need to be programmatically determinable.

**Comment by alastc** [comment]
The suggested definition in the original post doesn't appear to restrict it to pages.

**Comment by mbgower** [comment]
Thanks for your clarification, @alastc 
In some ways @mraccess77 is reinforcing my point:
> markers communicating page breaks need to be programmatically determinable

What we've ended up with is that markers that are 'programmatically determinable' need to be reachable. That's a lot lower bar.

> Such a wide scope could trigger quite a few things requiring navigation. E.g. Headings? Horizontal Rules?

The definition is still called "page break locators" :) But yes, my proposed definition gave a bit more scope.

If the group is intentionally tightly scoping this, I can live with it. I just question its value.

**Comment by mraccess77** [comment]
If something is programmatically determinable I think it will already have a better chance reachable - the real need from my perspective is making sure the page markers are programmatically determinable and not just text and also reachable.  But perhaps I don't use EPUBs enough -- it's just a common issue I find with Word and PDF alternatives from Bookshare and other sources.

**Comment by alastc** [comment]
> If the group is intentionally tightly scoping this, I can live with it. I just question its value.

If you remember there was another proposal for requiring that programmatic page-break markers are added, but the scope was really going outside of WCAG's remit. 

When we worked out where this really applies, I did go back to the ePub folk and check it was still worthwhile, and they thought it was. I think the the note in the [intent](https://w3c.github.io/wcag/understanding/page-break-navigation.html) section really sums it up.

**Comment by mraccess77** [comment]
Hi @alastc right now this seems to say if programmatic markers are provided... - can you give some examples of what would qualify as a programmatic marker?  This term is slightly different from our programmatically determinable terms so it's unclear what might be considered a programmatic marker.

**Comment by alastc** [comment]
If you have a look at the [technique](https://w3c.github.io/wcag/techniques/html/H99), there is a role of "doc-pagebreak" that I assume comes from epub.
